tests/vspec/test_overlay_on_instance/test_overlay_on_instance.py

```diff
@@ -20,25 +21,8 @@ def test_expanded_overlay(tmp_path):
     overlay1 = HERE / "overlay_1.vspec"
     overlay2 = HERE / "overlay_2.vspec"
     output = tmp_path / "out.json"
-    cmd = f"vspec export json -e my_id --pretty -u {TEST_UNITS}"
+    cmd = f"vspec export json -e my_id --pretty -u {TEST_UNITS} -q {TEST_QUANT}"
     cmd += f" --vspec {spec} -l {overlay1} -l {overlay2} --output {output}"
     subprocess.run(cmd.split(), check=True)
-
     expected = HERE / "expected.json"
     assert filecmp.cmp(output, expected)
-
-
-def test_expanded_overlay_no_type_datatype(tmp_path):
-    """
-    This test shows current limitation on type/datatype lookup.
-    We cannot do lookup on expanded names containing instances.
-    That would require quite some more advanced lookup considering instance declarations.
-    Not impossible, but more complex.
-    """
-    spec = HERE / "test.vspec"
-    overlay = HERE / "overlay_no_type_datatype.vspec"
```

Comment:
Is this file needed now when you have removed the test case?

Reply:
The file still contains another test case.